src/shared/components/svg/IconAdd.tsx

```diff
@@ -0,0 +1,18 @@
+export default function IconAdd() {
+  return (
+    <svg
+      width="40"
+      height="40"
+      viewBox="0 0 40 40"
+      fill="currentColor"
+      xmlns="http://www.w3.org/2000/svg"
+      xmlSpace="preserve"
+      className="MarvinBestSvg--icon-add"
```

Comment:
Ton icon devrait dans une viewbox de 30 sur 30 (comme sur la maquette).

```suggestion
      fill="currentColor"
      viewBox="0 0 30 30"
      xmlns="http://www.w3.org/2000/svg"
      xmlSpace="preserve"
      className="Icon--add"
```